README.md

```diff
@@ -5,4 +5,21 @@
 
 >focus: **web accessibility, Semantic HTML5, Presenting API data, Handling JSON data**
 
+**Installation**
+
+Prerequisites
+```
+npm install express
```

Comment:
When you install dependencies with `npm`, if you use the `--save` flag it will create a `package.json` file that contains the versions of the plugins used to make the site work. The `package.json` should be committed as part of the repo